website/public/js/controllers/groupsCtrl.js

```diff
@@ -173,6 +173,7 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
           member.petCount = Shared.countPets($rootScope.countExists(member.items.pets), member.items.pets);
           member.mountCount = Shared.countMounts($rootScope.countExists(member.items.mounts), member.items.mounts);
         $scope.profile = member;
+        $scope.e = member.items.gear.equipped;
```

Comment:
The relevant code is actually in `stats.jade`, which is used by both `profile.jade` and the member modal. The [initialization code](https://github.com/HabitRPG/habitrpg/blob/develop/website/views/shared/profiles/stats.jade#L42) is:
```
ng-init='g=Content.gear.flat;e=profile.items.gear.equipped'
```
I'm assuming this was done for brevity, because lines like this:
```
{{g[e.weapon].#{k} + g[e.armor].#{k} + g[e.head].#{k} + g[e.shield].#{k} || 0}}
```
would get extremely long otherwise.

All this PR does is override the `e` in stats.jade for the member modals, because the initialization isn't working for some reason.

So, to change the one-letter variable problem, `stats.jade` would need to be modified, and many of the lines would become noticeably longer. I can do that if you'd like, but I wanted to be sure that that's what you want.